ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.15__jenkins-ocp4.15-lp-interop.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
+  ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.19-openshift-4.13
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |-
+    FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
+    COPY . .
+    RUN chgrp -R 0 /go && \
+    chmod -R g=u /go
+    RUN mkdir -p /.cache
+    RUN chgrp -R 0 /.cache && \
+    chmod -R g=u /.cache
+  inputs:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.13:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
```

Comment:
Check OCP version

Reply:
see response here: https://github.com/openshift/release/pull/44988#discussion_r1376808230

if someone who knows this scenario well and can offer if this image can be updated please let us know.

Reply:
going to update and test this